Source/WebKit/Platform/cocoa/ExtensionCapability.h

```diff
@@ -31,21 +31,43 @@
 #include <wtf/RetainPtr.h>
 
 OBJC_CLASS BEProcessCapability;
+#if PLATFORM(IOS_SIMULATOR)
+OBJC_CLASS _SECapability;
+#endif
 
 namespace WebKit {
 
+#if PLATFORM(IOS_SIMULATOR)
```

Comment:
Should this be gated on an OS version too? Surely we want to use the proper API on recent SDKs?

Reply:
Yes, very good point, will fix!

Thanks for reviewing!